### DISCUSSION TITLE
Error when running DOSXYZnrc: "The job has failed"

### DISCUSSION THREAD

**Discussion by camilabustos1** [nrc-cnrc/EGSnrc#741] [Questions] [answered]
Hello everyone again 
I'm trying to create a phantom in DOSYXZnrc (voxel by voxel), which can be compiled but when I run it fails. I get this error. Thanks

![WhatsApp Image 2021-06-27 at 18 18 36](https://user-images.githubusercontent.com/85966458/123561370-febee180-d775-11eb-906d-38901c863cae.jpeg)
![WhatsApp Image 2021-06-27 at 18 19 17](https://user-images.githubusercontent.com/85966458/123561375-054d5900-d776-11eb-9ecb-4ed9ab8e293c.jpeg)
![WhatsApp Image 2021-06-27 at 18 19 40](https://user-images.githubusercontent.com/85966458/123561378-067e8600-d776-11eb-81f5-7ab60709540b.jpeg)

**Comment by rtownson** [comment, marked as answer]
It's reporting -1 voxels where it should be saying the number of voxels in the phantom along each axis. It seems like you might have misinterpreted the phantom dimensions section, could you share your input file and/or a screen shot of the phantom design window?

**Reply by kouadriaek** [reply]
bonjour
moi j ai cette erreur job failed quand je depasse 2.4 milliards de particules dans dosxyz

**Reply by rtownson** [reply]
@kouadriaek please make a new post about your issue, and include the .egslog file or the relevant errors (not just that the job has failed). Does it fail immediately or after some time?

**Reply by kouadriaek** [reply]
bonjour

no emmediately when i run 2000000000 particules in dosxyz if i do restart i have this message
